### PR TITLE
Modified hello-world upgrade test to upgrade from the version in Universe

### PR DESCRIPTION
**Pull request by susanxhuynh**
Modified the hello-world upgrade test to upgrade from the version in Universe. Added downgrade test.

### CODE REVIEW

File: frameworks/helloworld/integration/tests/test_upgrade.py [hunk 1]
```diff
@@ -28,23 +23,41 @@ def teardown_module(module):
...
 
 
-@pytest.mark.skip(reason="Currently fails...")
 @pytest.mark.sanity
-def test_upgrade():
+@pytest.mark.upgrade
+def test_upgrade_downgrade():
+    # Ensure both Universe and the test repo exist.
+    if len(shakedown.get_package_repos()['repositories']) != 2:
+        print('No test repo found.  Skipping test_upgrade_downgrade')
```
Comment (nickbp) [inline frameworks/helloworld/integration/tests/test_upgrade.py]: Maybe add a comment on when this is expected -- is this just expected to be a temporary situation?

Comment (susanxhuynh) [inline frameworks/helloworld/integration/tests/test_upgrade.py]: Thanks, @nickbp The use case is our Framework Test Suite, which runs only packages from Universe. It doesn't add a test repo like the PR jobs. I will add a comment.

**Comment by susanxhuynh** [comment]
retest this please

**Comment by susanxhuynh** [comment]
retest this please

**Pull request merged**